### ISSUE TITLE
Test can pass on postgresql REL_11_STABLE or later version

### ISSUE THREAD

**Issue by BaiShaoqi** [hlinnaka/pg_quota#1]
When make pg_quota on postgresql(install from source code on centos7)

Tried several release branches of postgresql, from REL9_4_STABLE to REL_11_STABLE.

pg_quota compile can only be sucessfully done on REL_11_STABLE or later.

**Comment by hlinnaka** [comment]
Yep, I developed this against then-current PostgreSQL master, which became REL_11_STABLE. It shouldn't be hard to make it work with older versions, though, if there's interest.

**Comment by BaiShaoqi** [comment]
Thanks for your reply, I understand.
